packages/api-clients/open-api/delegationApi.yml

```diff
@@ -37,6 +37,52 @@ tags:
       description: Find out more
       url: http://swagger.io
 paths:
+  /consumer/delegators:
+    parameters:
+      - $ref: "#/components/parameters/CorrelationIdHeader"
+      - in: query
+        name: delegatorName
+        schema:
+          type: string
+      - in: query
+        name: delegateId
+        required: true
+        schema:
+          type: string
+          format: uuid
+      - in: query
+        name: offset
+        required: true
+        schema:
+          type: integer
+          format: int32
+          minimum: 0
+      - in: query
+        name: limit
+        required: true
+        schema:
+          type: integer
+          format: int32
+          minimum: 1
+          maximum: 50
+    get:
+      tags:
+        - consumer
+      operationId: getConsumerDelegators
+      description: Retrieve delegators of a delegate with active delegation and agreement
+      responses:
+        "200":
+          description: Tenants retrieved
+          content:
+            application/json:
+              schema:
+                $ref: "#/components/schemas/CompactTenants"
```

Comment:
> See the comment above, this should probably return just a list of delegations and not a list of tenants. Usually, the tenant info for the delegator is then added with an enrichment in the BFF as we do for example when returning Compact Delegations https://vscode.dev/github.com/pagopa/interop-be-monorepo/blob/feature/incaricato/packages/backend-for-frontend/src/services/delegationService.ts#L47-L48
> 
> Here however we have a query on the tenant delegator name, so you already have to do the lookup on that table, so it's probably ok to return a delegation model that has also the delegatorName already

Maybe I should find a better name because this model doesn't return the whole tenant but only
```
{
   "id": "3fa85f64-5717-4562-b3fc-2c963f66afa2",
   "name": "Delegante Uno" 
}
```

Reply:
> See the comment above, this should probably return just a list of delegations and not a list of tenants. Usually, the tenant info for the delegator is then added with an enrichment in the BFF as we do for example when returning Compact Delegations https://vscode.dev/github.com/pagopa/interop-be-monorepo/blob/feature/incaricato/packages/backend-for-frontend/src/services/delegationService.ts#L47-L48
> 
> Here however we have a query on the tenant delegator name, so you already have to do the lookup on that table, so it's probably ok to return a delegation model that has also the delegatorName already

[Here](https://pagopaspa.slack.com/archives/C07U5M9SJ1Y/p1733905684079529) was discussed to implement this task with a dedicated route on the delegation process because doing an enrichment like we do in other cases is problematic